src/modules/SX126x.h

```diff
@@ -604,6 +604,17 @@ class SX126x: public PhysicalLayer {
     */
     int16_t setSyncWord(uint8_t* syncWord, uint8_t len);
 
+    /*!
+      \brief Sets FSK sync word in the form of array of up to 8 bytes.
```

Comment:
We should probably stress that setting a sync word here will override whatever was set by `setSyncWord`. Also, a note about ignoring LSB would be nice.

Reply:
I'm not sure if I do understand well, but overriding whatever was set by `setSyncWord `would be expected IMO. Should be mentined though. 

And yes, I was thinking about the way to describe the bits to be used, but I didn't find the right words so far. I will try to formulate that.